src/base.ts

```diff
@@ -79,38 +75,68 @@ export const resolveServiceUrl = (service: Service): string => {
   throw new Error(`Missing endpoint configuration for "${service}" service`)
 }
 
-const authorizationHeader = () => {
-  if (!apiClient.tokens.accessToken) {
-    return null
+const getAuthorizationHeader = (accessToken = null) => {
```

Comment:
```suggestion
const getAuthorizationHeader = (accessToken) => {
```

results in the same behavior :)

Reply:
I don't like undefined 😉 😂 
It's somewhat a mix between personal preference and readability whereas for me this indicates that the function might be called without the parameter and it does some handling in that case... Compared to not defaulting it will result in reading the function but that might be just me